StringBuilder.js

```diff
@@ -0,0 +1,57 @@
+// ES5 Way Inheritance
+import BuilderES5 from './BuilderES5.js'
+
+export default function StringBuilder (initialValue = '') {
+  StringBuilder._throwErrorIfValueNotAString(initialValue)
+  BuilderES5.prototype.constructor.call(this, initialValue)
```

Comment:
This line can be replaced with the following one:
BuilderES5.call(this, initialValue)

Reply:
replaced with the direct call as was suggested